tests/test_mariadb.py

```diff
@@ -208,3 +195,43 @@ def wait_for_server():
     assert rows and len(rows) == 2
     _, num, data = rows[-1].split()
     assert num == "100" and data == "abcdef"
+
+
+def test_mariadb_healthcheck_innodb_initialized(auto_container_per_test):
+    """
+    Test if InnoDB (storage engine) has completed initializing.
+
+    See: https://mariadb.com/kb/en/using-healthcheck-sh/#-innodb_initialized
```

Comment:
Converts this into a clickable link in the sphinx documentation
```suggestion
    See: `<https://mariadb.com/kb/en/using-healthcheck-sh/#-innodb_initialized>`_
```